slackmessage/send-to-channel-slack.yaml

```diff
@@ -15,13 +15,21 @@ spec:
     description: plain text message
   steps:
   - name: post
-    image: curlimages/curl:7.68.0
+    image: gcr.io/tekton-releases/dogfooding/catalog-web-requests:latest
```

Comment:
Please add a comment describing what this image is for, what it provides, etc.

I'm not actually sure we want to depend on it, since Tekton release infra might change to no longer require it, at which point it might get removed or stop getting updates. @afrittoli any thoughts?

Reply:
I agree @ImJasonH, infra images are built and maintained for infra, we probably should not use them in a catalog for our users.

Reply:
It's an image from the other PR on the plumbing repo specifically meant for catalog tasks. I would prefer to put the Dockerfile over here and make the catalog ci/cd build the image though. Is that possible?